js/index.js

```diff
@@ -40,3 +41,83 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let links = document.querySelectorAll("nav a");
+//console.log(links);
+links[0].innerHTML = siteContent['nav']['nav-item-1'];
```

Comment:
Good stuff! If you want to get fancy, you could do this:
`links.forEach((currentAnchor, i) => (currentAnchor.textContent = Object.values(siteContent.nav)[i]));`